store/storeResetCloner.js

```diff
@@ -0,0 +1,11 @@
+import cloneDeep from "lodash.clonedeep";
+export function storeResetCloner (store)  {
+  store.on('bookerInit', (storeContent) => {
+    return {intialStoreContent: cloneDeep(storeContent)};
+  });
+    
+  store.on('resetStore', ({intialStoreContent}) => {
+    const cloneInitialStoreContent = cloneDeep(intialStoreContent)
```

Comment:
```suggestion
    return cloneDeep(intialStoreContent);
```

És akkor az alatta lévő sor már nem is kell.